tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Version test for cookiecutter-python-template."""
+
+
+def test_version() -> None:
```

Comment:
This would be the tests for the cookiecutter python repo itself and not the project that was created using it. Hope that makes sense

Reply:
I have not added it yet and I think we need to add a test to verify versions for the cookiecutter template as it will be updated in the future in theory. I can add the tests later though. I have add a NotImplementedError() at the moment. Does that work?